common/Resources/ti.internal/extensions/node/stream.js

```diff
@@ -0,0 +1,124 @@
+import EventEmitter from './events';
+import util from './util';
+
+function Stream(opts) {
+	// FIXME: Can't call EventEmitter as a function!
+	this._eventsToListeners = {};
+	this._maxListeners = undefined;
+	// EventEmitter.call(this, opts);
+	// TODO: Provide more than an empty class?
+}
+Object.setPrototypeOf(Stream.prototype, EventEmitter.prototype);
+Object.setPrototypeOf(Stream, EventEmitter);
+// Use util.inherits?
+
+function Readable(options) {
+	if (!(this instanceof Readable)) {
+		return new Readable(options);
+	}
+
+	// TODO: readableState?
+
+	this.readable = true;
+
+	if (options) {
+		if (typeof options.read === 'function') {
+			this._read = options.read;
+		}
+		if (typeof options.destroy === 'function') {
+			this._destroy = options.destroy;
+		}
+	}
+
+	Stream.call(this);
+}
+util.inherits(Readable, Stream);
+Readable.prototype._destroy = function (err, cb) {
+	cb(err);
+};
+Readable.prototype._read = function (n) {
```

Comment:
<!--
  0 failure: 
  1 warning:  common/Resources/...
  
  
  DangerID: danger-id-Danger;
  File: common/Resources/ti.internal/extensions/node/stream.js;
  Line: 39;
-->

- :warning: common/Resources/ti.internal/extensions/node/stream.js line 39 – 'n' is defined but never used. Allowed unused args must match /^_.+/u. (no-unused-vars)